src/components/Dashboard/ThisWeek/NewsletterMenu/NewsletterMenu.test.tsx

```diff
@@ -197,6 +225,9 @@ describe('NewsletterMenu', () => {
                   },
                 },
               },
+              {
+                primaryPerson: null,
+              },
```

Comment:
I added this to show that the logic will filter out a value from the email list string if the person does not have an ```email```.